spec/DatabaseController.spec.js

```diff
@@ -361,6 +362,253 @@ describe('DatabaseController', function () {
       done();
     });
   });
+
+  describe('disableCollation', () => {
```

Comment:
Maybe it needs to be written calling `function()` instead of the short version like stated by @mtrezza to b10f?